.travis.yml

```diff
@@ -1,48 +1,57 @@
-language: python
 dist: xenial
-sudo: false
+language: python
+cache: pip
 python:
-- '3.5'
-- '3.6'
-- '3.7'
-- '3.8'
+  - 3.6
+  - 3.7
+  - 3.8
+  - pypy3
 env:
-- DJANGO="Django>=2.2,<2.3.0"     DB=postgresql DB_NAME=travis_ci_test
-- DJANGO="Django>=3.0"            DB=postgresql DB_NAME=travis_ci_test
-- DJANGO="Django>=2.2,<2.3.0"     DB=sqlite3 DB_NAME=db.sqlite3
-- DJANGO="Django>=3.0"            DB=sqlite3 DB_NAME=db.sqlite3
-- DJANGO="Django>=2.2,<2.3.0"     DB=mysql DB_NAME=mysql_db
-- DJANGO="Django>=3.0"            DB=mysql DB_NAME=mysql_db
-matrix:
-  exclude:
-  - python: '3.5'
-    env: DJANGO="Django>=3.0"            DB=postgresql DB_NAME=travis_ci_test
-  - python: '3.5'
-    env: DJANGO="Django>=3.0"            DB=sqlite3 DB_NAME=db.sqlite3
-  - python: '3.5'
-    env: DJANGO="Django>=3.0"            DB=mysql DB_NAME=mysql_db
-  fast_finish: true
+  - DJANGO=1.11     DATABASE=sqlite3
```

Comment:
We dropped Django 1.11 support in last release.

Reply:
OK, I just added that to the test matrix to see if the tests pass here. Django 1.11 is a LTS release which will still be supported by DSF until April 2020. 

I have no argument over support decisions, I'm just trying to offer improvements for the infrastructure. You decide what looks good to you!

Reply:
Okay, cool, let's keep Django 1.11 for now. We will drop it later in April.

Reply:
@nasirhjafri if the version is supported by the codebase and there is no sudden need to remove support I'd opt to support LTS releases as long as they are being maintained by DSF as that will make a lot of people running older code bases happy :)